### ISSUE TITLE
Maui template project: The "ResolvePackageAssets" task was not given a value for the required parameter "TargetFramework"

### ISSUE THREAD

**Issue by tedekeroth** [dotnet/maui#4244]
### Description

After creating a .NET Maui project from scratch, not changing, adding or removing anything at all, I get this error when trying to apply changes during execution of the default/template app on a real Android device:

```
The "ResolvePackageAssets" task was not given a value for the required parameter "TargetFramework"
C:\Program Files\dotnet\sdk\6.0.200-preview.22055.15\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets
```
I have to admit - I don't really know what this button is supposed to do, since I dont have to press that button for UI changes to show up on the device, that happens automatically after about 1 sec anyways.

I am using Visual Studio 2022 Preview version 17.1.0 Preview 3.

### Steps to Reproduce

1. Install VS 2022 Preview 17.1.0 Preview 3
2. Run maui-check, but skip the .NET Workload installation (it warns not to do that)
3. Create a new .NET Maui project (default template)
4. At this point, Visual Studio showed build errors, as discussed [in this SO thread](https://stackoverflow.com/questions/68046917/new-net-maui-app-project-throws-the-name-initializecomponent-does-not-exist), but VS was still able to build and launch the application both in emulator and on an Android device. After the successful build and run of the application, however, it no longer allowed me to build it again. I did the "workaround" descrived [in this answer](https://stackoverflow.com/a/69647457/178143) in the same SO thread, ad then the build errors went away.
5. Launch the app on real Android device
6. Click on "Apply Android resource changes and restart activity" button

At this point, the error above appears.

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android, I was *not* able test on other platforms

### Affected platform versions

Android 10, Win 10 10.0.19042 Build 19042

### Did you find any workaround?

No.

I did notice [this issue](https://github.com/dotnet/aspnetcore/issues/28473), where @mkArtakMSFT said:

> Your project is missing TargetFramework property declaration, as the error message points out.
> Consider adding the following in the PropertyGroup section in your project file:
> 
>     <TargetFramework>net5.0</TargetFramework>

And after I add that to the csproj file (but net6.0 instead), the error still appears after pressing the "Apply changes..." button.

This is not a big issue for me, but I thought I'd report it, maybe it is something that can be fixed.

**Comment by VincentBu** [comment]
Hi @tedekeroth , I didn't repro this issue. Just curious do you have the following workloads installed with Visual Studio?
![image](https://user-images.githubusercontent.com/44959937/158098212-1cf0503a-e6ea-4328-883b-81c447b5e1aa.png)

**Comment by tedekeroth** [comment]
Hello Vincent!
Nope, I dont have UWP. IS that needed for Maui? Is that required for Maui?
![image](https://user-images.githubusercontent.com/10088477/158224536-1e293f82-644a-44c3-9b61-931e53e0df29.png)

**Comment by VincentBu** [comment]
Well, I removed the UWP workload(it was required about one month ago) and still didn't see any error message. By the way, would you like to show your csproj file?

**Comment by Redth** [comment]
This should be fixed in 17.2 Preview 2.0.

Please open a new issue if you still have a problem there.
